angular.json

```diff
@@ -392,6 +392,44 @@
           }
```

Comment:
Adding the comment here, as I can't do it for non-modified files. You should also modify the following files:
- `projects/storefrontapp/tsconfig.app.prod.json`
- `projects/storefrontapp/tsconfig.server.json`
- `projects/storefrontapp/tsconfig.server.prod.json`
- `scripts/changelog.ts` - search for `const libraryPaths = {`
- `.github/api-extractor-action/api-extractor-for-branch.sh`:
```ts
PRODUCT_CONFIG_PATH="./../.github/api-extractor-action/api-extractor-configs/api-extractor.json"
if [ "$DIR" = "self" ] ; then
  PRODUCT_CONFIG_PATH="./.github/api-extractor-action/api-extractor-configs/api-extractor.json"
fi

...

# @spartacus/my-account
cp "$PRODUCT_CONFIG_PATH" ./dist/product-config/api-extractor.json
(
  cd ./dist/my-account && \
  api-extractor run --local --verbose
)
```

Reply:
What is needed in addition for the first 3 files? Because here I added productconfig already?

Reply:
You're right, nothing else to do there, my bad.